tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = integration-tests, flake8
```

Comment:
If you call this py36 then tox will look for Python 3.6. If you want to retain the name, you can use 'py36-integration' or similar.